x-pack/solutions/observability/plugins/apm/server/routes/suggestions/route.ts

```diff
@@ -39,6 +40,15 @@ const suggestionsRoute = createApmServerRoute({
     const coreContext = await context.core;
     const size = await coreContext.uiSettings.client.get<number>(maxSuggestions);
 
+    const hasUnset = await hasUnsetValueForField({
+      apmEventClient,
+      searchAggregatedTransactions,
+      serviceName,
+      fieldName,
+      start,
+      end,
+    });
+
```

Comment:
Could we run this in parallel with `getSuggestionsWithTermsEnum` and `getSuggestionsWithTermsAggregation`?